spanner_prober/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.19
+WORKDIR /go/src/github.com/GoogleCloudPlatform/grpc-gcp-go/spanner_prober/
+COPY . .
+RUN CGO_ENABLED=0 go build -a -installsuffix cgo -v -o /go/src/github.com/GoogleCloudPlatform/grpc-gcp-go/spanner_prober ./...
+
+FROM alpine:latest  
```

Comment:
I could not get this to work. Maybe some way to debug if all inputs are correct will help.